backend/substrapp/events/reactor.py

```diff
@@ -103,8 +103,8 @@ def on_function_event(payload):
             }
             (
                 # TODO switch to function.model_dump_json() as soon as pydantic is updated to > 2.0
-                build_image.si(**building_params).set(queue=builder_queue, task_id=function_key)
-                | save_image_task.si(**building_params).set(queue=WORKER_QUEUE, task_id=function_key)
+                build_image.si(**building_params).set(queue=builder_queue)
```

Comment:
we grab the id from the serialised function?

Reply:
yes